core/src/main/java/com/ritense/valtimo/service/CamundaProcessService.java

```diff
@@ -323,6 +337,70 @@ private void setProcessesExecutable(BpmnModelInstance bpmnModel) {
         );
     }
 
+    private void setToNullWhenServiceTaskExpressionIsEmpty(BpmnModelInstance bpmnModel) {
+        bpmnModel.getModelElementsByType(ServiceTask.class).forEach(task -> {
+            if (task.getCamundaType() == null
+                && task.getCamundaClass() == null
+                && task.getCamundaExpression() == null
+                && task.getCamundaDelegateExpression() == null) {
+                task.setCamundaExpression("${null}");
+                task.setCamundaAsyncAfter(true);
+            }
+        });
+    }
+
+    private void setToNullWhenSendTaskExpressionIsEmpty(BpmnModelInstance bpmnModel) {
+        bpmnModel.getModelElementsByType(SendTask.class).forEach(task -> {
+            if (task.getCamundaType() == null
+                && task.getCamundaClass() == null
+                && task.getCamundaExpression() == null
+                && task.getCamundaDelegateExpression() == null) {
+                task.setCamundaExpression("${null}");
+                task.setCamundaAsyncAfter(true);
+            }
+        });
+    }
+
+    private void setToCorrelateAllWhenMessageSendEventExpressionIsEmpty(BpmnModelInstance bpmnModel) {
+        Stream.of(IntermediateThrowEvent.class, EndEvent.class)
+            .flatMap(sendEventClass -> bpmnModel.getModelElementsByType(sendEventClass).stream())
+            .filter(sendEvent -> sendEvent.getId().matches("Event_[a-z0-9]{7}"))
+            .flatMap(sendEvent -> sendEvent.getChildElementsByType(MessageEventDefinition.class).stream())
+            .forEach(event -> {
+                if (event.getCamundaType() == null
+                    && event.getCamundaClass() == null
+                    && event.getCamundaExpression() == null
+                    && event.getCamundaDelegateExpression() == null) {
+                    String messageName = event.getMessage() == null ? "MY_MESSAGE" : event.getMessage().getName();
+                    event.setCamundaExpression(
+                        "${correlationService.sendMessageToAll(\"" + messageName + "\",execution)}"
+                    );
+                }
+            });
+    }
+
+    private void setToRoleUserWhenUserTaskCandidateGroupsIsEmpty(BpmnModelInstance bpmnModel) {
+        bpmnModel.getModelElementsByType(UserTask.class).forEach(userTask -> {
+            if (userTask.getId().matches("Activity_[a-z0-9]{7}")
+                && userTask.getCamundaCandidateUsers() == null
+                && userTask.getCamundaCandidateGroups() == null) {
+                userTask.setCamundaCandidateGroups("ROLE_USER");
+            }
+        });
+    }
+
+    private void setTo60SecondsWhenTimerIsEmpty(BpmnModelInstance bpmnModel) {
```

Comment:
I think this goes too far, a timer event really should be configured explicitly by the user. Forgetting to put in a 4 week duration can't mean you get one minute by default.